src/main/java/org/terracotta/build/plugins/PackagePlugin.java

```diff
@@ -58,6 +53,7 @@
 @SuppressWarnings("UnstableApiUsage")
 public abstract class PackagePlugin implements Plugin<Project> {
 
+  public static final String COMPONENT_NAME = "package";
```

Comment:
`JvmConstants.JAVA_COMPONENT_NAME` is gone after 8.4 so we take the opportunity to switch to our own component name.